ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerDemoController.swift

```diff
@@ -9,6 +9,9 @@ import UIKit
 // MARK: DrawerDemoController
 
 class DrawerDemoController: DemoController {
+
```

Comment:
How about changing the name here to shouldConfirmDrawerDismissal?

Reply:
updated